types/scroll-to-element/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for scroll-to-element 2.0
+// Project: https://github.com/willhoag/scroll-to-element
+// Definitions by: Kirill Kvashonin <https://github.com/kirillurgant>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface Options {
+    offset?: number;
+    align?: 'top' | 'middle' | 'bottom';
+    ease?: string;
+    duration?: number;
+}
+
+declare function scrollToElement(selector: string | HTMLElement | Element, options?: Options): void;
+
+export default scrollToElement;
```

Comment:
This needs `export =` instead of `export default`. Refer to https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default